lib/logstash/outputs/elasticsearch/elasticsearch-template-es5x.json

```diff
@@ -20,24 +20,23 @@
           "match" : "*",
           "match_mapping_type" : "string",
           "mapping" : {
-            "type" : "string", "index" : "analyzed", "omit_norms" : true,
-            "fielddata" : { "format" : "disabled" },
+            "type" : "text", "norms" : false,
             "fields" : {
-              "raw" : {"type": "string", "index" : "not_analyzed", "ignore_above" : 256}
+              "keyword" : { "type": "keyword" }
```

Comment:
Yes, but we need `"norms" : false,` to be set, which ES does not by default. That's why I had to explicitly add this rule for strings.